tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Handles logging functionality.
+"""
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority, publisher_id, msg):
+    """
+    Function used for logging errors/output/info.
+    Args:
+        log_priority : Priority of the Log Message
+        publisher_id : Id of publisher
+        msg          : Message to be logged
+    """
+    try:
```

Comment:
Try Except block is not required everywhere. It was used at the places which were invoked before Namespace NS was created.